index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>GOGOmedia - Recruitment Task</title>
+
+	<!-- Google Fonts -->
+	<link href="https://fonts.googleapis.com/css?family=Rubik:300,400,500,700&amp;subset=latin-ext" rel="stylesheet">
+
+	<!-- Font Awesome Icons -->
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.0/css/all.css" integrity="sha384-Mmxa0mLqhmOeaE8vgOSbKacftZcsNYDjQzuCOm6D02luYSzBG8vpaOykv9lFQ51Y" crossorigin="anonymous">
+
+	<!-- Style CSS -->
+	<link rel="stylesheet" href="css/normalize.css">
+	<link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+	<!-- header -->
+	<header class="header-container">
+			<section class="container clearfx">
+				<div class="header-container__logo">
+					<a href="#"><img src="images/logo.png"></a>
+				</div>
+			</section>
+			<div class="banner-container">
+				<section class="container clearfx">
+					<h1 class="banner-container__title">Heading</h1>
+					<p class="banner-container__text">Lorem ipsum dolor sit amet, consect etur adipiscing elit.</p>
+					<div class="box-container">
```

Comment:
Nazwy klas są trochę nielogiczne. Mieszasz dużo różnych rzeczy razem i to jest nieczytelne